eng/pipelines/mgmt-auto-release.yml

```diff
@@ -1,32 +1,35 @@
 trigger: none
-
 pr: none
 
-schedules:
```

Comment:
I created a daily schedule on the `yml` definition instead of leaving it here.

Reply:
Hi @scbedd,
Could you please provide the daily schedule on the build path? I can't find it, thank you.